extensions/wikia/PortableInfobox/styles/PortableInfobox.scss

```diff
@@ -81,6 +81,35 @@
 	text-align: center; // center smaller image inside infobox
 }
 
+.pi-image-gallery {
+	position: relative;
+
+	&::after {
+		background-image: url("/extensions/VisualEditor/modules/ve-mw/ui/styles/images/icons/gallery-ltr-white.svg");
```

Comment:
I know it's a mock, though please add a TODO to remove this coupling with VE styles